src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -7077,6 +7077,16 @@ public object VisitParenExpression(ParenExpressionAst parenExpressionAst)
         {
```

Comment:
@rjmholt, your last commit had 1 failures in `PowerShell-CI-linux`
Experimental Feature: && and || operators - Feature-Enabled.Does not flatten nested values in assignment
```powershell
Expected 2, but got 3.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Operators/PipelineChainOperator.Tests.ps1: line 172
172:         $x.Count | Should -Be 2
```